jvm/hermes/src/main/jni/JJSIValue.h

```diff
@@ -4,9 +4,11 @@
 #include <jsi/jsi.h>
 #include <fbjni/fbjni.h>
 #include <fbjni/ByteBuffer.h>
+#include "RuntimeScope.h"
 
 using namespace facebook::jni;
 using namespace facebook::jsi;
+using namespace std;
```

Comment:
If we're going to be `using namespace std`, we should actually remove all the `std::` references.

Probably would help to set up a linter.